lib/salus/scanners/report_pom_xml.rb

```diff
@@ -0,0 +1,38 @@
+require 'salus/scanners/base'
+
+# Report Java library usage
+
+module Salus::Scanners
+  class ReportPomXml < Base
+    def run
+      shell_return = run_shell("bin/parse_pom_xml #{@repository.path_to_repo}/pom.xml", chdir: nil)
+
+      if !shell_return.success?
+        report_error(shell_return.stderr)
```

Comment:
Could you call `report_stderr` here too?
I forgot the exact reason that both `report_error` and `report_stderr` are needed, but that's what other scanners do.  I think one of the prints to the screen or something.

Reply:
You may want to check on how `shell_return.success?` is defined. It might look for a specific status code or if you explicitly wrote error to stderr or something.